addons/medical_treatment/stringtable.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="AFMED">
+    <Package name="Medical_Treatment">
+        <!--
+            MEDICATIONS
+        -->
+        <Key ID="STR_AFMED_Medical_Treatment_Apap_DisplayName">
+            <English>Apap pastille</English>
+            <Polish>Apap tabletka</Polish>
+        </Key>
+        <Key ID="STR_AFMED_Medical_Treatment_Apap_Desc_Short">
+            <English>Used to combat light pain experiences. No side effects.</English>
+            <Polish>Lekkie działanie przeciwbólowe. Brak efektów ubocznych.</Polish>
+        </Key>
+        <!--
+        <Key ID="STR_AFMED_Medical_Treatment_Apap_Desc_Use">
+            <English></English>
+            <Polish></Polish>
+        </Key>
+        -->
+        <Key ID="STR_AFMED_Medical_Treatment_Inject_Apap">
+            <English>Eat Apap</English>
+            <Polish>Zjedz Apap</Polish>
+        </Key>
+        <Key ID="STR_AFMED_Medical_Treatment_Injecting_Apap">
+            <English>Eating Apap...</English>
+            <Polish>Zjadanie Apapu...</Polish>
```

Comment:
As this can also be administered to someone else, maybe "Administering Apap"/"Podawanie Apapu"

Reply:
Good idea, already did that.